snap/hooks/install

```diff
@@ -33,10 +33,30 @@ for service in security-api-gateway security-secret-store core-command config-se
     fi
 done
 
-# also handle config-seed/device-virtual/application.properties
-if [ ! -f "${SNAP_DATA}/config/config-seed/res/properties/device-virtual/application.properties" ]; then
-    mkdir -p "${SNAP_DATA}/config/config-seed/res/properties/device-virtual"
-    cp "${SNAP}/config/config-seed/res/properties/device-virtual/application.properties" "${SNAP_DATA}/config/config-seed/res/properties/device-virtual/application.properties"
+# handle device-mqtt driver configuration
+cp ${SNAP}/config/device-mqtt/res/configuration-driver.toml "${SNAP_DATA}/config/device-mqtt/res/configuration-driver.toml"
+
+# handle device-random device profile
+cp ${SNAP}/config/device-random/res/device.random.yaml "${SNAP_DATA}/config/device-random/res/device.random.yaml"
+
+# also handle java services' application.properties
+for jsvc in device-virtual; do
+    if [ ! -f "${SNAP_DATA}/config/config-seed/res/properties/$jsvc/application.properties" ]; then
+        mkdir -p "${SNAP_DATA}/config/config-seed/res/properties/$jsvc"
+        cp "${SNAP}/config/config-seed/res/properties/$jsvc/application.properties" "${SNAP_DATA}/config/config-seed/res/properties/$jsvc/application.properties"
+        # also replace SNAP_DATA and SNAP_COMMON in the application files
+        sed -i -e "s@\$SNAP_COMMON@$SNAP_COMMON@g" -e "s@\$SNAP_DATA@$SNAP_DATA@g" "${SNAP_DATA}/config/config-seed/res/properties/$jsvc/application.properties"
+    fi
+done
+
+# create support-rulesengine directories for templates/rules
```

Comment:
Isn't rules-engine till Java? If so, I guess we do need to keep the afore-mentioned (pun intended) **for** loop, and add rules-engine to the list of svcs?

Reply:
Actually, after review, we haven't yet added rules-engine to master (it's yet another contribution to delhi that we didn't forward port at the time), so we should just remove all refs to rules-engine in this file and keep the **for** loop after all.

Reply:
This was leftover from my merge resolution with cherry-picking, I removed it.